src/ui/MobileMenu/index.tsx

```diff
@@ -13,4 +13,5 @@ const MobileMenu: React.FC<MobileBottomSheetProps> = (props: MobileBottomSheetPr
   );
 };
 
+export { MobileMenu, MobileContextMenu, MobileBottomSheet };
 export default MobileMenu;
```

Comment:
Do we need to export in both ways?
 
```
export { MobileMenu };
export default MobileMenu;
```

Reply:
I think it's better for covering both case, we can deprecate one way in the future.
But until that time, we will keep both way.